CompanyApi/Controllers/EmployeeApi.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using CompanyApi.Model;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace CompanyApi.Controllers
+{
+    [ApiController]
+    [Route("Companies/{companyID}/Employees")]
+    public class EmployeeApi : ControllerBase
+    {
+        [HttpPost]
+        public ActionResult<Company> AddNewEmployee(string companyID, Employee employee)
+        {
+            var company = FakeDatabase.GetCompanyByID(companyID);
+
+            if (company == null)
+            {
+                return NotFound();
+            }
+
+            if (company.Employees == null)
+            {
+                company.Employees = new List<Employee>();
+            }
```

Comment:
This part might be redundant since Company is always initialized with new List<Employee>();